packages/grafana-ui/src/components/RefreshPicker/RefreshPicker.tsx

```diff
@@ -72,6 +76,7 @@ export class RefreshPicker extends PureComponent<Props> {
           variant={variant}
           icon={isLoading ? 'fa fa-spinner' : 'sync'}
           aria-label={selectors.components.RefreshPicker.runButton}
+          style={style}
```

Comment:
```suggestion
          fullWidth={true}
```

If we make the `ToolbarButton` to grow and take all of the unused space and set the constraint (width) on the `ButtonGroup` using `cx` and classNames. I think that would be a more elegant way of doing this and less prone to cause problems else where.

Reply:
So in the `ButtonGroup` you should be able to pass a className lite below and it will only be applied when `width` has a truthy value.

```ts
      <ButtonGroup
        className={cx('refresh-picker', {
          refreshPickerWidth: !!width,
        })}
      >
```